docs/src/getting-started/how-it-compares.md

```diff
@@ -1,42 +1,43 @@
 # How it Compares
 
-Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between it and the Fuel indexer. Generally, the biggest conceptual differences between the two are as follows:
+- Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between The Graph and the Fuel indexer.
```

Comment:
Since this is a introductory paragraph, we should remove the bullet point here and just make it a regular sentence.